core-bundle/src/Resources/contao/languages/en/modules.xlf

```diff
@@ -299,11 +299,11 @@
       <trans-unit id="FMD.template.1">
         <source>Generates a module to insert a custom template.</source>
       </trans-unit>
-      <trans-unit id="FMD.rssReader.0">
-        <source>RSS reader</source>
+      <trans-unit id="FMD.feed_reader.0">
+        <source>Feed reader</source>
       </trans-unit>
-      <trans-unit id="FMD.rssReader.1">
-        <source>Adds an RSS feed to the page</source>
+      <trans-unit id="FMD.feed_reader.1">
+        <source>Adds an RSS, Aom or JSON feed to the page</source>
```

Comment:
```suggestion
        <source>Adds an RSS, Atom or JSON feed to the page</source>
```

Reply:
Fixed in [97503bb](https://github.com/contao/contao/pull/4847/commits/97503bb7c765f681a44912e05d5dc4d37fa99974)